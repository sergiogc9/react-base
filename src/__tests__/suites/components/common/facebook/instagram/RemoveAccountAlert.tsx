import { shallow } from 'enzyme';
import React from 'react';

import RemoveAccountAlert from '@src/components/common/facebook/instagram/RemoveAccountAlert';

let onAcceptMock: jest.Mock;
let onCancelMock: jest.Mock;

describe('<RemoveInstagramAccountAlert />', () => {
	let wrapper: any;

	beforeEach(() => {
		onAcceptMock = jest.fn();
		onCancelMock = jest.fn();
		wrapper = shallow((<RemoveAccountAlert
			onAccept={onAcceptMock}
			onCancel={onCancelMock}
		></RemoveAccountAlert>));
	});

	it("Test RemoveAccountAlert component snapshot", () => {
		expect(wrapper.html()).toMatchSnapshot();
	});
});
