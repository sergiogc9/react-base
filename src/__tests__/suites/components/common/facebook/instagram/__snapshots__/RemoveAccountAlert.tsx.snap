// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RemoveInstagramAccountAlert /> Test RemoveAccountAlert component snapshot 1`] = `"<span class=\\"md-dialog-container md-overlay md-overlay--active md-pointer--hover launchmetrics-dialog\\" tabindex=\\"-1\\"><div aria-labelledby=\\"launchmetrics-dialog\\" id=\\"removeInstagramAccountAlertDialog\\" role=\\"dialog\\" class=\\"md-paper md-paper--5 md-dialog md-dialog--centered md-background--card\\"><h2 id=\\"removeInstagramAccountAlertDialog-title\\" class=\\"md-title md-title--dialog\\"><div class=\\"dialog-close\\"><button id=\\"launchMetricsDialogClosBtn\\" type=\\"button\\" class=\\"md-btn md-btn--flat md-btn--text md-pointer--hover md-text md-inline-block btn--no-background\\"><div class=\\"md-ink-container\\"></div><i class=\\"md-icon material-icons\\">clear</i></button></div><div class=\\"dialog-header md-grid\\"><div class=\\"dialog-logo\\"><i class=\\"md-icon material-icons\\">warning</i></div><div class=\\"dialog-title\\">account.instagram_accounts.dialog.title</div></div></h2><section class=\\"md-dialog-content\\"><span aria-hidden=\\"true\\"></span><div class=\\"dialog-container\\"><div class=\\"dialog-content\\"><div class=\\"dialog-text\\">account.instagram_accounts.dialog.content</div></div></div></section><footer class=\\"md-dialog-footer md-dialog-footer--inline dialog-footer md-grid \\"><button id=\\"launchmetricsDialogCancelBtn\\" type=\\"button\\" class=\\"md-btn md-btn--flat md-btn--text md-pointer--hover md-text md-inline-block md-btn--dialog btn--grey btn--no-background\\"><div class=\\"md-ink-container\\"></div>form.cancel</button><button id=\\"launchmetricsDialogAcceptBtn\\" type=\\"button\\" class=\\"md-btn md-btn--flat md-btn--text md-pointer--hover md-text md-inline-block md-btn--dialog btn--green btn--no-background\\"><div class=\\"md-ink-container\\"></div>account.instagram_accounts.dialog.confirm</button></footer></div></span>"`;
